neuralprophet/forecaster.py

```diff
@@ -720,6 +720,24 @@ def predict(self, df, decompose=True, raw=False):
         forecast = pd.DataFrame()
         for df_name, df_i in df.groupby("ID"):
             dates, predicted, components = self._predict_raw(df_i, df_name, include_components=decompose)
+            if self.config_missing.drop_missing and self.max_lags > 0:
```

Comment:
could we make this into a function?
Why do we need to call this in predict?

Reply:
Done! Pls see PR #747